src/index.js

```diff
@@ -21,37 +22,68 @@ function questions() {
             ` File with ${answers.name}.json already exists, try again! `
           )
         );
-        questions();
+        start();
       } else {
-        prompt([
-          {
-            type: "input",
-            name: "name",
-            message: "What is your name?",
-          },
-          {
-            type: "select",
-            name: "type",
-            message: "Your profile type?",
-            choices: ["Personal", "Community"],
-          },
-          {
-            type: "input",
-            name: "bio",
-            message: "Add a short bio about yourself",
-          },
-        ])
-          .then((answers) => {
-            createUser(GHUsername, answers);
-          })
-          .catch((err) => {
-            console.log(err);
-          });
+        checkUser(GHUsername).then((result) => {
+          if (result === true) {
+            questions(GHUsername);
+          } else {
+            console.log(
+              chalk.bgRed.bold(
+                ` User with username ${GHUsername} does not exist, try again! `
```

Comment:
```suggestion
                ` User with username ${GHUsername} does not exist on GitHub, try again! `
```

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix